LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2021 [COMPANY NAME]
+Copyright (c) 2021 DevOps for Defense
```

Comment:
```suggestion
Copyright (c) 2023 AaC Project Contributors
```

Reply:
Updated here and on home page where original reference was made.